client/state/users/suggestions/reducer.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	USER_SUGGESTIONS_RECEIVE,
+	USER_SUGGESTIONS_REQUEST,
+	USER_SUGGESTIONS_REQUEST_SUCCESS,
+	USER_SUGGESTIONS_REQUEST_FAILURE,
+} from 'state/action-types';
+
+/**
+ * Returns the updated requests state after an action has been dispatched. The
+ * state maps site ID keys to whether a request for user suggestions is in progress.
+ *
+ * @param  {Object} state   Current state
+ * @param  {Object} action  Action payload
+ * @return {Object}         Updated state
+ */
+export function requesting( state = {}, action ) {
+	switch ( action.type ) {
+		case USER_SUGGESTIONS_REQUEST:
+		case USER_SUGGESTIONS_REQUEST_SUCCESS:
+		case USER_SUGGESTIONS_REQUEST_FAILURE:
+			return {
+				...state,
+				[ action.siteId ]: USER_SUGGESTIONS_REQUEST === action.type,
+			};
+	}
+
+	return state;
+}
+
+/**
+ * Returns the updated items state after an action has been dispatched. The
+ * state maps site ID keys to an array of user suggestions.
+ *
+ * @param  {Object} state   Current state
+ * @param  {Object} action  Action payload
+ * @return {Object}         Updated state
+ */
+export function items( state = {}, action ) {
```

Comment:
@gwwar I've added schema validation [here](https://github.com/Automattic/wp-calypso/pull/9969/commits/a2cc6cc0d8f82277b3a4e72d6ddb70e6dee28615). The data structure for user suggestions is as per the below:

![suggestions-data-structure](https://cloud.githubusercontent.com/assets/1190420/21287602/3e0b79d0-c43d-11e6-8313-8bec342e1721.png)

Cheers!